README.md

```diff
@@ -74,6 +74,12 @@ pip3 install https://github.com/containers/podman-compose/archive/main.tar.gz
 brew install podman-compose
 ```
 
+### Generate binary using docker/podman locally
+This script will download the repo, generate the binary using [this Dockerfile](https://github.com/containers/podman-compose/blob/main/Dockerfile.md) and let the binary in the directory where you called this script.
+```bash
+sh -c "$(curl -sSL https://raw.githubusercontent.com/containers/podman-compose/main/scripts/download_podman-compose.sh)"
```

Comment:
I think this should better named as `build_podman-compose.sh`, because we're not just downloading. Also add a mention that the script will call sudo in case of docker-based build because this is rather sensitive to some people.

Reply:
Hmmm, yeah, you are probably right, but since there is already a script called `generate_binary_using_dockerfile.sh`, having one with "build" could lead to confusion.

What do you think about naming it "build_and_download_podman-compose.sh" ?

Reply:
Makes sense. But maybe  download_and_build_podman-compose.sh ? So that reading left to right the sequence of events is correct.

Reply:
I agree with you. I just pushed the changes :smile: